demos/unity_states/unity/Assets/Scripts/BallStateDataSourceSynchronizer.cs

```diff
@@ -0,0 +1,49 @@
+using UnityEngine;
+using TP = Teleportal.Client;
+using States = Teleportal.Client.Contract.Properties.States;
+using RSharp;
+
+public class BallStateDataSourceSynchronizer : MonoBehaviour
+{
+    [SerializeField] private BallStateDataSourcePlatform srcPlatform;
+    [SerializeField] private BallStateDataSourceUnity srcUnity;
+    private bool readyToSynchronize = false;
+
+    void Awake()
+    {
+        if ((null != this.srcPlatform) && (null != this.srcUnity))
+        {
+            this.readyToSynchronize = true;
+        }
+        else
+        {
+            UnityEngine.Debug.LogError("Platform or Unity data source is not assigned!");
+        }
+    }
+
+    void Update()
+    {
+        ApplyUnityStateToPlatformBaseline();
```

Comment:
nit: same opinion here about the trivial helper function

Reply:
Ditto previous response about lifecycle function clutter